bleak/backends/bluezdbus/manager.py

```diff
@@ -10,12 +10,12 @@
 import contextlib
 import logging
 import os
+from collections import defaultdict
```

Comment:
Move to line 10 to keep imports sorted?

Reply:
CI lint job is passing, so I think this is already sorted correctly according to `isort`.